.github/CODEOWNERS

```diff
@@ -5,6 +5,7 @@
 /src/docs/platforms/android/ @getsentry/team-mobile
 /src/docs/platforms/cocoa/ @getsentry/team-mobile
 /src/docs/platforms/flutter/ @getsentry/team-mobile
+/src/docs/platforms/react-native/ @getsentry/team-mobile
```

Comment:
Everything from `/src/docs/platforms/` moved to `/src/platforms/` in #2048.

Going to fix the paths here.

Reply:
Thanks!